models/doctype/Item/Item.js

```diff
@@ -111,10 +111,17 @@ export default {
         }
       },
     },
+    {
+      fieldname: 'itemCode',
+      label: 'Item Code',
+      fieldtype: 'Int',
+      placeholder: 'Item Code',
```

Comment:
`itemCode` at least in ERPN serves as a unique identifier, this purpose is already served in FB by `name` (_Item Name_).

Any particular reason for adding this in? Also if it's required why _Int_? Imo should be alphanumeric, _Data_ seems apt.

Reply:
Ah okay got it. Might as well add in the conditional required before merge?